plugin.video.embycon/addon.xml

```diff
@@ -15,7 +15,7 @@
     <platform>all</platform>
     <language>en</language>
     <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
-    <forum>https://emby.media/community/index.php?/topic/57016-embycon-kodi-addon/</forum>
```

Comment:
Can you change the license field to "GPL-2.0"?  (spdx identifier). It's not mandatory but rather a recommendation.

Reply:
Changing to "GPL-2.0-only"